filament/src/IndirectLight.cpp

```diff
@@ -197,10 +197,17 @@ FIndirectLight::FIndirectLight(FEngine& engine, const Builder& builder) noexcept
 void FIndirectLight::terminate(FEngine& engine) {
     if (FEngine::CONFIG_IBL_USE_IRRADIANCE_MAP) {
         FEngine::DriverApi& driver = engine.getDriverApi();
-        driver.destroyTexture(mIrradianceMapHandle);
+        driver.destroyTexture(getIrradianceMap());
```

Comment:
There's a weird name mismatch in the variables:
builder->mReflectionsMap
mIrradianceTexture
getIrradianceMap()

Either call them all texture or all map, but not a mix of both.

Reply:
Amended the commit -- now using the "Texture" suffix for methods that return textures and using "HwHandle" for methods that return a HwHandle.  Avoiding Map.